docs/api/instruments/ni/virtualbench.rst

```diff
@@ -0,0 +1,32 @@
+##################################
```

Comment:
Line lengths for the title markers (i.e. `#` and `*`) should match the length of the content.

Reply:
done